### DISCUSSION TITLE
Multiple-Start Threading

### DISCUSSION THREAD

**Discussion by Lester444** [kachurovskiy/nanoels#79] [Q&A] [answered]
Hello Maxim.

Quick question: 
Is there a way to do multiple-start threads using ELS_H2? I was hoping to make some knurling on a knob by combining multiple-starts with left-hand & right-hand threads.

**Comment by kachurovskiy** [comment, marked as answer]
Hi @Lester444, yes, I've added multi-start functionality in https://github.com/kachurovskiy/nanoels/commit/9b95523736a2602adf484427337052fe25ae3a17 and documentation on how to use it in https://github.com/kachurovskiy/nanoels/commit/13db8cbacb546664e910aeee10140a306e5fb0c4

See the updated documentation at https://github.com/kachurovskiy/nanoels/blob/main/h2/README.md#operating-the-els

Please let me know if it works for you.

**Reply by Lester444** [reply]
Wow that was fast!!!

Give me a few days to try it out. My hardware is wired differently and so I need to add a switch for multi-start.

Will surely let you know how it turns out.
